internal/reporter/github.go

```diff
@@ -289,7 +289,13 @@ func reportToGitHubComment(headCommit string, rep Report) *github.PullRequestCom
 			msgPrefix,
 			rep.Problem.Text,
 		)),
-		Line: github.Int(reportLine),
+	}
+
+	if reportLine < 0 {
```

Comment:
I think it's better to remove `&& report.Problem.Severity == checks.Fatal` from https://github.com/cloudflare/pint/blob/main/internal/reporter/bitbucket.go#L273